internal/sirius/get_deputy_events_test.go

```diff
@@ -2,55 +2,62 @@ package sirius
 
 import (
 	"bytes"
+	"github.com/ministryofjustice/opg-sirius-supervision-deputy-hub/internal/mocks"
+	"github.com/stretchr/testify/assert"
 	"io"
 	"net/http"
 	"net/http/httptest"
 	"testing"
-
-	"github.com/ministryofjustice/opg-sirius-supervision-deputy-hub/internal/mocks"
-	"github.com/stretchr/testify/assert"
+	"time"
 )
 
+func AmendDateForDST(date string) string {
+	if time.Now().Local() != time.Now() {
```

Comment:
Do you need this condition? Can't you just apply `FormatDateAndTime` to all dates in the test and the formatter will handle whether it needs to be amended or not?